apps/marginfi-v2-docs/src/app/the-arena/page.mdx

```diff
@@ -1,70 +1,120 @@
 import { ImageComponent } from '~/components/ImageComponent'
 
 import CreateBankExample from '~/images/docs/create-bank-example.png'
+import SupplyExample from '~/images/docs/supply-example.png'
+import TradeExample from '~/images/docs/trade-example.png'
+import PortfolioExample from '~/images/docs/portfolio-example.png'
 
 export const metadata = {
   title: 'The Arena User Guide',
-  description: 'A User Guide to Making Leveraged Trades with Longtail Assets',
+  description: 'The Arena is a permisionless lending market that allows users to long and short any token on Solana',
 }
 
 # The Arena: A User Guide to Degeneracy
 
-For the first time on Solana, take leveraged long and short positions on longtail assets (e.g., memecoins, or assets with little liquidity) and permissionlessly create your own isolated banks for any token, using any collateral. {{ className: 'lead' }}
+For the first time on Solana, take leveraged long and short positions on any asset (e.g., memecoins, celebrity coins, major market cap tokens) and permissionlessly create your own isolated banks for any token, using any collateral. {{ className: 'lead' }}
 
 ---
 
-## Trade Memecoins with Leverage {{ tag: '', label: '' }}
-
-
-
 ## Permissionless Banks {{ tag: '', label: '' }}
 
 Historically, the creation of isolated banks has been controlled by the marginfi team. marginfi has utilized isolated banks to allow users to borrow and lend certain assets without their liquidity and borrow-lend activity affecting the main liquidity pool - hance the term "isolated."
 
-However, with The Arena, the marginfi team is opening up the ability to create isolated banks to everyone. Now, **you can permissionlessly create isolated banks for any token you want, using any collateral you want** directly through <Button href="https://www.thearena.trade/" variant="text"><>The Arena app</></Button>, the marginfi CLI, or the marginfi TS SDK.  Because permissionless banks are isolated banks, **the trading activity that occurs within any given permissionless bank does not impact marginfi's main liquidity pool**. 
+However, with The Arena, the marginfi team is opening up the ability to create isolated groups to everyone. Now, **you can permissionlessly create isolated banks for any token you want, using any collateral you want** directly through <Button href="https://www.thearena.trade/" variant="text"><>The Arena app</></Button>, the marginfi CLI, or the marginfi TS SDK.  Because permissionless banks are isolated banks, **the trading activity that occurs within any given permissionless bank does not impact marginfi's main liquidity pool**. 
```

Comment:
mentioning "you can now create isolated groups" and the sentence after "you can create isolated banks" sounds a bit confusing